scripts/kubescape-backlog-bridge/main.go

```
@@ -0,0 +1,434 @@
+		}
+		if err := checkHydration(items, *minSample); err != nil {
+			return fmt.Errorf("cve input: %w", err)
+		}
+		themes = append(themes, deriveCVE(items)...)
+	}
+
+	return report(themes, out)
+}
+
+func readList(path string) ([]item, error) {
+	raw, err := os.ReadFile(path)
```

golangci-lint (2.12.2)

[medium] G304: Potential file inclusion via variable

(gosec)